generator/generators/RustGenerator.py

```diff
@@ -59,6 +59,14 @@ def _escape_rust_keywords(s):
                 return "{}_".format(s)
             return s
 
+        def _format_numeric_literal(value):
```

Comment:
Since sometimes this function is not doing anything, I would propose to rename it to `try_format_numeric_literal` to avoid confusion, why we are formatting consts of any type as numeric literals.